Resources/Other-Sources/it.lproj/Localizable.strings

```diff
@@ -0,0 +1,44 @@
+"DifferentTemperatureFormat" = "È %@ %@ rispetto %@%@.";
+"PrecipitationChanceFormat" = "%@ possibilità di %@ %@.";
+"SameTemperatureFormat" = "La temperatura %@ %@ rispetto a %@%@.";
+
+"Hotter" = "molto più caldo";
+"Warmer" = "più caldo";
+"Cooler" = "più freddo";
+"Colder" = "molto più freddo";
+"Same" = "è stabile";
+
+"Today" = "oggi";
+"Yesterday" = "ieri";
+"Tonight" = "questa notte";
+"LastNight" = "la scorsa notte";
+"ThisMorning" = "questa mattina";
+"YesterdayMorning" = "ieri mattina";
+"ThisAfternoon" = "questo pomeriggio";
+"YesterdayAfternoon" = "ieri pomeriggio";
+
+"NoneRain" = "";
+"SlightRain" = ", con bassa probabilità di pioggia";
+"GoodRain" = ", con alta probabilità di pioggia";
+"NoneSnow" = ", nessuna possibilità di nevicate";
+"SlightSnow" = ", con bassa probabilità di nevicate";
+"GoodSnow" = ", con alta probabilità di nevicate";
+
+"Updated %@ at %@" = "Aggiornato il %@ alle %@";
```

Comment:
Rather than using the format string to do the localization, what do you think about calling this `UpdatedAt` and then also update the english localization to be `"UpdatedAt" = "Updated %@ at %@"` ?